crates/query-engine/sql/src/sql/helpers.rs

```diff
@@ -506,6 +522,29 @@ pub fn select_rows_as_json(
     select
 }
 
+/// Turn each row of a query result into a json object.
+///
+/// ```sql
+/// SELECT row_to_json(<table_alias>) AS <column_alias>
+/// FROM <query> as <table_alias>
+/// ```
+///
+/// - `row_to_json` takes a row and converts it to a json object.
+/// - `coalesce(<thing>, <otherwise>)` returns `<thing>` if it is not null, and `<otherwise>` if it is null.
```

Comment:
This part is probably copy paste from the with_default version right? :)

Reply:
Right - I've pruned it moer now. Thanks